data/hai/hai reveal 4 middle.txt

```diff
@@ -1508,27 +1508,115 @@ mission "Hai Reveal [A14] A Time of Trouble"
-			`	The next several hours pass slowly as they do their work, support in the Cabinet is gained, and the data sets collected are assessed for potentially viable priorities. Towards the end of it, you're not really in the loop anymore. The mechanisms of government simply don't include you, but after a while Batu comes to find you. "Come on, there's a meeting between the President and the Elders. You should come and join the witnesses."`
-			`	It takes a short while, but after some doing they manage to set up a clear channel dedicated to this particular call and you are all gathered into a room with large wall-mounted displays. You knew it was coming, and yet you're still a little surprised to see before you the President of the Republic and the Chancellor of the Free Worlds in remarkable clarity and surprisingly low latency. The windows behind the Chancellor suggest they were awoken in the middle of the night for this, but you wouldn't guess it from looking at them.`
-			`	Teeneep begins, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
+				goto questions
+			label begin
+			`	A few hours later, they manage to set up a clear channel dedicated to the meeting in a room with large wall-mounted displays. You knew it was coming, and yet you're still a little surprised to see before you the President of the Republic and the Chancellor of the Free Worlds in remarkable clarity and surprisingly low latency. The windows behind the Chancellor suggest they were awoken in the middle of the night for this, but you wouldn't guess it from looking at them.`
+			`	After a few seconds, you realize that three galactic leaders are waiting for you to begin the proceedings. As you've never been a galactic negotiator, you improvise the best you can.`
+			choice
+				`	"The tragedy we see around us can become a future of peace. Would the Hai representative please present us the problem we must solve together?"`
+					goto formality
+				`	"Teeneep, please explain the problem to the human leaders. I'm sure we can come to a compromise that will suit everyone."`
+			`	Teeneep responds, "Thank you for mediating these proceedings, Captain <last>."`
+			`	Both human leaders seem put off by the level of familiarity you just expressed, but Teeneep looks as if you took a world of stress off her shoulders.`
```

Comment:
This sequence feels slightly gratuitous.

Reply:
I removed the "Both human leaders..." paragraph.